client/components/FormBuilder.jsx

```diff
@@ -0,0 +1,70 @@
+/* eslint react/prop-types: 0 */
+/* eslint react/destructuring-assignment: 0 */
+/* eslint react/no-array-index-key: 0 */
+
+import React from 'react';
+import {
+  Avatar, Button, Container, Box, Typography, CssBaseline, Grid, TextField,
+} from '@material-ui/core';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import { Alert } from '@material-ui/lab';
+import { useStyles } from '../styles/useStyles';
```

Comment:
since you're only using two styles from this component I wonder if it would be better to setup a separate useStyles() inside of this component.  Kind of like David Lee did with navbar.